README.md

```diff
@@ -46,6 +46,8 @@ You can use NixOS shell, which will have the Git-LFS, Go and others
 
 `nix-shell`
 
+Note: for mac users, you will need to comment out the line `pkgsCross.aarch64-multiplatform.buildPackages.gcc` in order for the build to work
+
```

Comment:
FYI, this is not needed anymore as the part of the build that originally needed this now uses docker. ref: https://github.com/tinkerbell/boots/blob/06204de91d2afb509f6ca9422180c56265f34fa9/rules.mk#L97

Reply:
This was brought in from the merges to master. I've now rebased and its all gone.